js/adapt-contrib-spoor.js

```diff
@@ -14,15 +14,15 @@ define([
     //Session Begin
 
         initialize: function() {
-            this.listenToOnce(Adapt, "configModel:dataLoaded", this.onConfigLoaded);
-            this.listenToOnce(Adapt, "app:dataReady", this.onDataReady);
+            this.listenToOnce(Adapt, {
+                'offlineStorage:prepare': this.onPrepareOfflineStorage,
+                'app:dataReady': this.onDataReady
```

Comment:
considering `onDataReady` is now a one-line, single-use method, I don't think it needs its own method any more, could you this change to:
```js
this.listenToOnce(Adapt, {
    "configModel:dataLoaded": this.onConfigLoaded,
    "app:dataReady": function() {
        Adapt.wait.for(adaptStatefulSession.initialize.bind(adaptStatefulSession));
    }
});
```
while you're here?

Reply:
No problem.